### PR TITLE
UI :- Change explore left panel to old style ui

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Change explore a left panel to old-style UI
- Added Clear All button and remove specific deselect button
- remove the entity type from the left panel filter.

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx [hunk 1]
```diff
@@ -99,6 +99,16 @@ const FacetFilter: React.FC<FacetFilterProps> = ({
 
   return (
     <>
+      <div className="sidebar-my-data-holder mt-2 mb-3">
+        <Button
+          className="text-primary cursor-pointer p-0"
+          disabled={isEmpty(filters)}
+          type="link"
+          onClick={() => onClearFilter({})}>
+          Clear All
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx:108]: localization

File: openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FilterContainer.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const FilterContainer: FunctionComponent<FilterContainerProp> = ({
 
   return (
     <label
-      className="filter-group tw-justify-between tw-mb-2.5 tw-cursor-pointer"
+      className="filter-group tw-justify-between m-b-xs tw-cursor-pointer"
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FilterContainer.tsx:45]: Can we remove tailwind styling from here and use existing styles?

File: openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx [hunk 3]
```diff
@@ -180,7 +181,7 @@ const FacetFilter: React.FC<FacetFilterProps> = ({
                 )}
                 {aggregationsPageSize[aggregationKey] > 5 && (
                   <p
-                    className="link-text tw-text-xs"
+                    className="link-text text-xs tw-text-left"
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx:184]: same here. Can we remove the tailwind styling

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment! also have you tested all the scenario right?

File: openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FilterContainer.tsx (hunk 2 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FilterContainer.tsx]: @aniketkatkar97 updated!

**Review by aniketkatkar97 — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx]: @ShaileshParmar11  yes, it is tested.

File: openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx [hunk 4]
```diff
@@ -99,20 +101,30 @@ const FacetFilter: React.FC<FacetFilterProps> = ({
...
+      <div className="sidebar-my-data-holder mt-2 mb-3">
+        <Button
+          className="text-primary cursor-pointer p-0"
+          disabled={isEmpty(filters)}
+          type="link"
+          onClick={() => onClearFilter({})}>
+          {t('label.clearAll')}
+        </Button>
+      </div>
+      <hr className="m-y-xs" />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/facetfilter/FacetFilter.tsx:113]:
```suggestion
      <Divider className="m-y-xs" />
```

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**